e2e/app/component/sidebar-content.ts

```diff
@@ -61,10 +64,46 @@ export default class SidebarContent extends  HelpSidebar {
     }
    
     // get the annotations name displaying on the sidebar panel
-    async getAnnotationsName(): Promise<string> {
-      const xpath = '//*[contains(normalize-space(text()), "Annotations")]/following::div[4]';
-      const element = BaseElement.asBaseElement(this.page, await this.page.waitForXPath(xpath, {visible: true}));
-      return element.getTextContent();
+    async getAnnotationsName(annotationFieldName:string): Promise<string> {
+    const selector = this.getFieldNameXpath(`${annotationFieldName}`);
+    const annotationfieldLabel = await this.extractFieldNameText(`${selector}`);
+    console.log (`annotationfieldLabel: ${annotationfieldLabel}`);
+    return annotationfieldLabel;
     }
 
+    // get the xpath of the respective annotation field label
+    private getFieldNameXpath(fieldName: string): string {
+      return `//*[contains(normalize-space(text()), "Annotations")]/following::div[contains(normalize-space(), "${fieldName}")]`;
+    }
+
+    // extract only the annotation field name
+    private async extractFieldNameText(selector: string): Promise<string> {
+      const elemt = await this.page.waitForXPath(`${selector}`, {visible: true});
+      const textContent = await getPropValue<string>(elemt, 'textContent');
+      const regex = new RegExp(/(aoutest)-\d+/);
+     return (regex.exec(`${textContent}`))[0];
+    }
+
+     // click on the Annotations EDIT button to edit the annotations field name
+     async getAnnotationsEdit(): Promise<Button>{
+      return Button.findByName(this.page, {name: LinkText.Edit});
+    }
+
+     // text area
+     async getAnnotationsTextArea(): Promise<Textarea> {
+      const selector = `${this.getXpath()}//*[contains(normalize-space(text()), "Annotations")]/following::textarea`;
```

Comment:
single quote. not backtick.

Reply:
Hi @aweng98 could you confirm about having a single quote here and not backtick?

Reply:
I'm wrong. backtick is right here. I think I didn't see `${this.getXpath()`